src/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Post } from '../types';
+import UserInfo from '../UserInfo/UserInfo';
+import CommentList from '../CommentList/CommentList';
+import './PostInfo.scss';
+
+type Props = {
+  post: Post;
+};
+
+const PostInfo:React.FC<Props> = ({ post }) => {
+  return (
+    <>
+      <UserInfo user={post.user ? post.user : null} />
```

Comment:
use conditional rendering, not just add condition to props

https://reactjs.org/docs/conditional-rendering.html